library/Zend/Mail.php

```diff
@@ -497,15 +497,16 @@ public function getPartCount()
      * characters.
      *
      * @param  string $value
+     * @param  int $linelength
      * @return string
      */
-    protected function _encodeHeader($value)
+    protected function _encodeHeader($value, $linelength = Zend_Mime::LINELENGTH)
```

Comment:
Is this for 2.2.1?
create new method with additional parameter where only encryption will be done. Call this new method in current function. And there won't be duplication of isPrintable() in _formatAddress() method.

Reply:
Done